tensorflow/contrib/tensorrt/convert/convert_graph.cc

```diff
@@ -910,7 +909,7 @@ tensorflow::Status ConvertAfterShapes(ConversionParams& params) {
              : EngineInfo::EngineType::TRTStatic);
     curr_engine.cached_engine_batches = params.cached_engine_batches;
     curr_engine.maximum_cached_engines = params.max_cached_engines;
-    StrAppend(&curr_engine.engine_name, "my_trt_op_", t);
+    StrAppend(&curr_engine.engine_name, "TRTEngineOp_", t);
```

Comment:
I believe this will most of existing tests especially the python tests. Any reason to change it?

Reply:
Done